panther_driver/README.md

```diff
@@ -1,6 +1,6 @@
 # Panther driver
```

Comment:
- the whole file is in different style compared to the rest of the README files.
- `use_pdo` is not documented 
- `/joint_states` in array might use convention with `fl_wheel_joint` instead of `Front left`. Or just a note that all arrays follow this order.
- `can_interface` parameter might have string "name of the socket CAN interface"
- `motor_torque_constant` parameter might have string "constant used to estimate torque"
- `power_factor` string is not descriptive at all
- parameters are not in alphabetical order
- CAN bus section can be completely removed

Reply:
Power factor will be modified later.  Apart from that, it is ok.